### PR TITLE
Some further updates for DR9, and merge security updates.

### PR DESCRIPTION
**Pull request by geordie666**
This PR is mainly to merge in the security updates from #120 for the DR9 pages before we progress too much further. It also includes some updates for DR9:
- A new acknowledgment that includes the copyright for the viewer and is aligned with the language of the acknowledgment used at NOIRLab.
- Changing the phrase "NSF's OIR Lab"  to "NOIRLab" everywhere.
- Publications through early October, 2020.
- A handful of other minor notes and fixes.

### CODE REVIEW

**Comment by geordie666** [comment]
@weaverba137: I thought I'd better merge my branch with master and push my WIP branch to ensure that I don't break your recent security updates. 

The merge was relatively painless as your chosen approach made it easy for me to reestablish all of my links to the DR9 pages. Thanks!

I don't think you need to review this for page content, but you _should_ check that your updated `jQuery` and `Bootstrap` versions work with this branch.

File: conf.py [hunk 1]
```diff
@@ -16,7 +16,7 @@
 # BASE_URL = "https://getnikola.com/"
 BLOG_EMAIL = "no-reply@legacysurvey.org"
 BLOG_DESCRIPTION = "Legacy Survey"
-LOGO_URL = "/files/logo_tiny.jpg"
+LOGO_URL = "/files/logo_very_tiny.jpg"
```
Comment (weaverba137) [inline conf.py:19]: This one I'm concerned about, I had to tweak the CSS corresponding to this file after updating the jQuery + Bootstrap framework.  We might need to just test this on several browsers.

**Review by weaverba137 — COMMENTED** [review]
Aside from the issue with the navbar, this all looks good. If this branch is deployed on your test site, we can just ask people to hit that with various browsers & OS.

File: conf.py (hunk 1 above)
Comment (weaverba137) [reply conf.py:19]: PS, the custom.css file assumes a specific `height` for the image in the navbar, but `logo_tiny` and `logo_very_tiny` do not have the same height.

Comment (geordie666) [reply conf.py]: I reverted back to `logo_tiny.jpg` and cleaned up the toolbar accordingly. My only reason for using `logo_very_tiny.jpg` was to put an extra column in the toolbar, but that was a stylistic choice and hardly critical.

**Comment by geordie666** [comment]
@weaverba137: I just reverted back to something closer to the original navbar. If this now works on your side, feel free to merge this PR.

I like the idea of hitting [the test site](http://faraday.uwyo.edu/~admyers/scratch/legacysurvey/output/) with various browsers. I've tried that with:
```
Safari 14.0 (15610.1.28.1.9, 15610)
Firefox 82.0.2 (64-bit)
Chrome 86.0.4240.111 (Official Build) (x86_64)
```
on my Mac laptop and it looks fine. But, it would certainly be useful for more people to stress test the general appearance of the site.

**Comment by weaverba137** [comment]
@geordie666, my own systems are not particularly different from yours, but I can fire up a Firefox on Linux test.  Have you put the word out on decam-chatter for people to test?

**Comment by geordie666** [comment]
@weaverba137: I haven't put the word out. Do you think it's critical for this PR, or is it something we can wait and do when we have the final website (with final page updates, etc.) in place?

**Comment by weaverba137** [comment]
If you're satisfied, then I think it's fine to merge.  The nice thing about Bootstrap+jQuery it is safe to assume that it will be supported on all modern browsers.

**Pull request merged**